### PR TITLE
Allow starting connection only once

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts [hunk 1]
```diff
@@ -27,4 +27,72 @@ describe("Connection", () => {
             done();
         }
     });
+
+
```
Comment (BrennanConroy) [inline client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts]: nit: `\n`

File: client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts [hunk 2]
```diff
@@ -27,4 +27,72 @@ describe("Connection", () => {
...
+    it("cannot start a running connection", async (done) => {
+        let options: ISignalROptions = {
+            httpClient: <IHttpClient>{
+                get(url: string): Promise<string> {
+                    if (url.indexOf("/negotiate") >= 0) {
+
```
Comment (BrennanConroy) [inline client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts]: nit: `\n`

**Comment by BrennanConroy** [comment]
:shipit:

**Comment by moozzyk** [comment]
Merged as 7944be712fadd3d206a36ab7810254f977c352f4

**Pull request closed without merging**